easy-ui-react/src/Button/Button.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ReactElement, ReactNode, useRef } from "react";
+import { IconProps } from "Icon";
+import { useButton } from "@react-aria/button";
+import { classNames, variationName } from "../utilities/css";
+
+import styles from "./Button.module.scss";
+
+export type ButtonColor =
+  | "primary"
+  | "secondary"
+  | "success"
+  | "warning"
+  | "neutral"
+  | "support"
+  | "inverse";
+export type ButtonAppearance = "filled" | "outlined" | "link";
+export type ButtonSize = "sm" | "md";
+
+export type ButtonProps = {
+  /** Button color */
+  color?: ButtonColor;
+  /** Button appearance */
+  appearance?: ButtonAppearance;
```

Comment:
how would you feel about using `variant` instead of `appearance`? i know we used `appearance` previously but from research it seems `variant` is a little bit more ubiquitous/versatile in component libraries. hoping we can standardize on a convention to use throughout